frontend/src/components/Form.tsx

```diff
@@ -109,9 +109,9 @@ export const Form: React.FunctionComponent<FormProps> = ({
   console.log(wavelengthUnitWatch);
   React.useEffect(() => {
     if (wavelengthUnitWatch === "u.nm") {
-      setSimulateslitUnit(true);
+      setSimulateSlitUnit(true);
```

Comment:
Could we instead maybe create a hook for `useWavenumber(true|false)` or `useWavelength(true|false)` or even `wavelengthUnit(string)`? Right now `setSimulateSlitUnit` has a built-in assumption as to whether we default to wavenumber or wavelength for setting the label, but we could remove this dependency by instead checking for whether or not we're using wavelength or wavenumber.

Reply:
@arunavabasu-03